theseus/utils/examples/pose_graph/dataset.py

```diff
@@ -135,7 +135,7 @@ def read_2D_g2o_file(
 
                 sel = [0, 3, 5]
                 weight = th.Variable(
-                    torch.from_numpy(np.array(tokens[6:], dtype=np.float64)[sel])
+                    torch.from_numpy(np.array(1, tokens[6:], dtype=np.float64)[sel])
```

Comment:
Why do we need this change?

Reply:
Explicitly adding a batch dimension removes a warning from `DiagonalCostWeight` that the shape is ambiguous.